appcircle_ios_testinium_upload_app/1.0.0/component.yaml

```
@@ -0,0 +1,58 @@
+platform: iOS
+buildPlatform: ObjectiveCSwift, ReactNative
+displayName: Testinium Upload App
+description: "Enables uploading mobile applications to the Testinium platform."
+docsUrl: https://docs.appcircle.io/workflows/common-workflow-steps/testinium-upload-app
+requiredComponents: "appcircle_ios_build_sign"
+precedingComponents:
+followingComponents:
+webUrl: https://github.com/appcircleio/appcircle-testinium-upload-app-component
+repoUrl: https://github.com/appcircleio/appcircle-testinium-upload-app-component.git
+commit: 2eedbf2
+inputs:
+- key: "AC_TESTINIUM_APP_PATH"
+  defaultValue: "$AC_APK_PATH"
```

Default value is wrong for iOS. We made this in appcenter distribute -> https://github.com/appcircleio/appcircle-workflow-components/blob/develop/appcircle_ios_appcenter_distribute/1.0.2/component.yaml

```suggestion
  defaultValue: "$AC_EXPORT_DIR"
```
or we can use the $AC_TESTINIUM_APP_PATH